src/createServer.js

```diff
@@ -1,3 +1,32 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+/* eslint-disable no-console */
+const http = require('http');
+const { validateReq } = require('./validateReq');
+const { prepareResponse } = require('./prepareResponse');
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    const normalizedUrl = new URL(req.url, `http://${req.headers.host}`);
+    const textToConvert = normalizedUrl.pathname.slice(1);
+    const caseToConvert = normalizedUrl.searchParams.get('toCase');
```

Comment:
move it to the separate function
```suggestion
    const [textToConvert, caseToConvert] = ...(...);
```